gammapy/scripts/image_pipe.py

```diff
@@ -285,7 +285,8 @@ def make_exposure(self, obs_id, spectral_index=2.3, for_integral_flux=False):
         xpix_coord_grid, ypix_coord_grid = exposure.coordinates(coord_type='pix')
         # calculate pixel offset from center (in world coordinates)
         coord = pixel_to_skycoord(xpix_coord_grid, ypix_coord_grid, exposure.wcs, origin=0)
-        center = SkyCoord.from_pixel(self.header["NAXIS1"] / 2., self.header["NAXIS2"] / 2., exposure.wcs)
+        center = exposure.center()
```

Comment:
@adonath 
Do you agree now with the test I added that the center() method of the Skymap is ok here?